reactioncommerce/reaction
# Fix translation issues in PDP

Resolves #2676

I had to manually update the non-English translations from master. This should work now on `marketplace`

### Steps to test
1. Login as admin on `marketplace` branch
1. Select product and go to the PDP
1. Switch your language to German
1. Click on the option edit icon to open the side panel
1. Observe that fields here are translated.
1. Switch to any other language and observe that side panel fields are appropriately translated
